src/main/external-resources/renderers/BlackBerry-PlayBook-KalemSoftMP.conf

```diff
@@ -39,7 +39,7 @@ Supported = f:wmv                 m:video/x-ms-wmv
 Supported = f:wav    m:audio/wav        n:6   a:dts|lpcm     s:48000
 Supported = f:wav    m:audio/wav        n:2                  s:48000
 Supported = f:mp3    m:audio/mpeg       n:2
-Supported = f:aac    m:audio/x-m4a      n:2   a:(?!alac).+
+Supported = f:m4a    m:audio/x-m4a      n:2   a:(?!alac).+
```

Comment:
@Sami32 I'm not sure I follow you. It's a negative lookahead, so it means "anything *but* alac". That said, it's easy to see when editing the renderer configurations that most are just copy-hasted from other configurations without actually checking anything.

The use of mime-types on the supported lines should also be cleaned up. Mime-type only needs to be specified if the renderer requires a *different* mime-type than what we use as standard for that format. That means that e.g specifying is meaningless because ```HTTPResource``` says:
```java
	public static final String AUDIO_M4A_TYPEMIME = "audio/x-m4a";
```